project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Stores active status visibility is broken
<img width="614" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/22ab4676-b95b-41f7-aa56-1514b65afe70">